subgraph.template.yaml

```diff
@@ -1636,3 +1732,68 @@ templates:
         - event: SuperAdminTransferred(indexed address,indexed address,address[])
           handler: handleIAdminACLV0SuperAdminTransferred
       file: ./src/mapping-v3-core.ts
+
+  - name: IGenArt721CoreContractV3_Engine_Flex_Template
```

Comment:
optional minor nit: can we move this above the AdminACLV0_Template, just to keep the core contract templates all together?